core/src/test/java/org/apache/iceberg/TestMergeAppend.java

```diff
@@ -147,25 +135,22 @@ public void testEmptyTableAppendFilesAndManifest() throws IOException {
             table.newAppend().appendFile(FILE_C).appendFile(FILE_D).appendManifest(manifest),
             branch);
 
-    Assert.assertNotNull("Should create a snapshot", committedSnapshot);
+    assertThat(committedSnapshot).isNotNull();
     V1Assert.assertEquals(
         "Last sequence number should be 0", 0, table.ops().current().lastSequenceNumber());
     V2Assert.assertEquals(
         "Last sequence number should be 1", 1, table.ops().current().lastSequenceNumber());
-    Assert.assertEquals(
-        "Should create 2 manifests for initial write",
-        2,
-        committedSnapshot.allManifests(table.io()).size());
+    assertThat(committedSnapshot.allManifests(table.io())).hasSize(2);
 
     long snapshotId = committedSnapshot.snapshotId();
 
     ManifestFile committedManifest1 = committedSnapshot.allManifests(table.io()).get(0);
     ManifestFile committedManifest2 = committedSnapshot.allManifests(table.io()).get(1);
 
     if (formatVersion == 1) {
-      Assertions.assertThat(committedManifest2.path()).isNotEqualTo(manifest.path());
+      assertThat(committedManifest2.path()).isNotEqualTo(manifest.path());
```

Comment:
same as above. Which actual/expected is correct here? because in one of the other files you changed it to be the other way around

Reply:
Confirmed `manifest.path()` is always "expected". I'll fix other opposite tests